app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -43,6 +56,13 @@ def validate_import_vm
     Gem::Version.new(api_version) >= Gem::Version.new('4')
   end
 
+  def submit_configure_imported_vm_networks(userid, vm_id)
```

Comment:
TBH this method should neither be on a Vm nor on the Manager but on a VmImporter class. But we havent embraced something like ServiceObjects - https://github.com/ManageIQ/manageiq/issues/12902
So, you'll find both in our codebase. 🤷‍♂️ I'd say keep it here, so its at least in a `VmImport` module...

But `userid` should be `user_id`, to be consistent.